docs/user-guide.md

```diff
@@ -429,7 +429,7 @@ services:
 
 If the Docker Compose file has a volume specified for a service, the Deployment (Kubernetes) or DeploymentConfig (OpenShift) strategy is changed to "Recreate" instead of "RollingUpdate" (default). This is done to avoid multiple instances of a service from accessing a volume at the same time.
 
-If the Docker Compose file has service name with `_` in it (eg.`web_service`), then it will be replaced by `-` and the service name will be renamed accordingly (eg.`web-service`). Kompose does this because "Kubernetes" doesn't allow `_` in object name.
+If the Docker Compose file has service name with `_` or `.` in it (eg.`web_service` or `web.service`), then it will be replaced by `-` and the service name will be renamed accordingly (eg.`web-service`). Kompose does this because "Kubernetes" doesn't allow `_` in object name nor metadata name (rises "DNS-1035 label must consist of lower case alphanumeric characters").
```

Comment:
Isn't object name and metadata name the same thing?

Reply:
Others LGTM

Reply:
Right! 

I made the distinction because the original documentation refered to a problem with the "spec" part of the kube file, but at the end, metadata is just another Kubernetes object so I guess it can be simplified to "object name" :)